boards/csm64f02.json

```diff
@@ -0,0 +1,31 @@
+{
+  "build": {
+    "core": "esp8266",
+    "extra_flags": "-DESP8266 -DARDUINO_ARCH_ESP8266 -DARDUINO_ESP8266_CSM64F02",
+    "f_cpu": "80000000L",
+    "f_flash": "40000000L",
+    "flash_mode": "qio",
+    "ldscript": "eagle.flash.2m.ld",
+    "mcu": "esp8266",
+    "variant": "nodemcu"
+  },
+  "connectivity": [
+    "wifi"
+  ],
+  "frameworks": [
+    "arduino",
+    "simba",
+    "esp8266-rtos-sdk",
+    "esp8266-nonos-sdk"
+  ],
+  "name": "CHIPSEA CSM64F02",
+  "upload": {
+    "maximum_ram_size": 81920,
+    "maximum_size": 2097152,
+    "require_upload_port": true,
+    "resetmethod": "nodemcu",
+    "speed": 115200
+  },
+  "url": "http://www.esp8266.com/wiki/doku.php?id=esp8266-module-family",
```

Comment:
Should probably be http://www.chipsea.com/en/wifi/CSM64F02.html, not the espressif documentation